examples/Dockerfile.web

```diff
@@ -0,0 +1,77 @@
+# Define the Container registry for our base images, as these are not stored on Dockerhub.

```

Comment:
Many of the comments from Dockerfile.service apply equally to Dockerfile.web.

Reply:
So do the fixes.